### DISCUSSION TITLE
Broker $SYS topic

### DISCUSSION THREAD

**Discussion by Navorski** [dotnet/MQTTnet#1667] [Q&A]
I wrote a handler when a client connects to the server, i would like the server to send some system information to the client

$SYS/version
$SYS/uptime
$SYS/clients/active = x
...

But how can i send this information to the newly connected client alone? since all the other connected clients don't need an update.

Or what would be the correct way to handle this?

**Comment by gfoidl** [comment]
* don't use MQTT for this, as the "brokering" isn't needed
* use a client-unique topic for this -- i.e. the client sends a Guid with the connect, then the server publishes and the special topic (that uses that Guid as part of the topic) the desired info

**Reply by Navorski** [reply]
I understand but i got the idea because of a common Mosquitto server, such a server is also doing that.
![image](https://user-images.githubusercontent.com/7352877/216019491-775b5fc3-e79c-4731-b860-73ed5589b165.png)
